go.mod

```diff
@@ -0,0 +1,73 @@
+module github.com/Mellanox/nvidia-k8s-ipam
+
+go 1.20
+
+require (
+	github.com/go-logr/logr v1.2.4
+	github.com/spf13/cobra v1.7.0
+	k8s.io/api v0.26.4
+	k8s.io/apimachinery v0.26.4
```

Comment:
The latest controller-runtime package (0.14.6) is incompatible with client-go 1.27.

We don't need the latest and greatest features from client-go 1.27, it is fine to use 1.26 and then upgrade when needed.